pkg/api/query/v1.go

```diff
@@ -683,6 +692,35 @@ func NewRulesHandler(client rules.UnaryClient, enablePartialResponse bool) func(
 	}
 }
 
+// NewExemplarsHandler creates handler compatible with HTTP /api/v1/exemplars [link-to-be-added]
+// which uses gRPC Unary Rules API.
+func NewExemplarsHandler(client exemplars.UnaryClient, enablePartialResponse bool) func(*http.Request) (interface{}, []error, *api.ApiError) {
+	ps := storepb.PartialResponseStrategy_ABORT
+	if enablePartialResponse {
+		ps = storepb.PartialResponseStrategy_WARN
+	}
+
+	return func(r *http.Request) (interface{}, []error, *api.ApiError) {
+		_, err := cortexutil.ParseTime(r.FormValue("start"))
+		if err != nil {
+			return nil, nil, &api.ApiError{Typ: api.ErrorBadData, Err: err}
+		}
+		_, err = cortexutil.ParseTime(r.FormValue("end"))
+		if err != nil {
+			return nil, nil, &api.ApiError{Typ: api.ErrorBadData, Err: err}
+		}
+
+		req := &exemplarspb.ExemplarsRequest{
```

Comment:
Looks like some of my original changes are missing in your rebase. Can you please check https://github.com/yeya24/thanos/commit/1b6abfcb35e2a7153a9dabda921926a7f72b8c7f#diff-f14ed7ae9f9be7d76dea77aa31304b22de00af7e5f08720a7b6e75f721caf24aR700-R711?

Reply:
Oh, I missed this one. Thanks for pointing it out.